docs/docsite/rst/user_guide/complex_data_manipulation.rst

```diff
@@ -136,6 +136,29 @@ Another way is to avoid adding elements to the list in the first place, so you c
           - "bar"
 
 
+Custom Fileglob Based on a Variable
+-----------------------------------
+
+This example uses `Python argument list unpacking <https://docs.python.org/3/tutorial/controlflow.html#unpacking-argument-lists>`_ to create a custom list of fileglobs based on a variable.
+
+.. code-block:: YAML+Jinja
+  :caption: Using fileglob with a list based on a variable.
+
+    - hosts: all
+      vars:
+        mygroups
+          - prod
+          - web
+      tasks:
+        - name: Copy a glob of files based on a list of groups
+          copy:
+            src: "{{ item }}"
+            dest: "/tmp/{{ item }}"
+          loop: '{{ q("fileglob", *globlist) }}'
+          vars:
+            globlist: '{{ mygroups | map("regex_replace", "^(.*)$", "files/\1/*.conf" | list }}'
```

Comment:
```suggestion
            globlist: '{{ mygroups | map("regex_replace", "^(.*)$", "files/\1/*.conf") | list }}'
```

Reply:
^ da missing parens ... sorry, thought it was obvious once pointed out ... forget not everyone has Jinja2 parser integrated neural pathways.

Reply:
Gah. Sorry I couldn't find that.